Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM stefanscherer/node-windows as builder
+
+COPY bin bin
+COPY src src
+COPY .babelrc .babelrc
+COPY .eslintrc.json .eslintrc.json
+COPY package.json package.json
+COPY package-lock.json package-lock.json
+
+RUN npm install
+RUN npm run build
+
+COPY boot.cmd boot.cmd
+
+FROM builder
+
+COPY --from=builder nodejs nodejs
```

Comment:
dit is niet echt nodig toch? Zou verwachten dat 'ie het node-windows image ook in dit image gebruikt?